### ISSUE TITLE
ListView: Touchpad usage behaviour does not have an animation

### ISSUE THREAD

**Issue by Murmele** [slint-ui/slint#10421]
### Bug Description

When using a ListView and [this PR](https://github.com/slint-ui/slint/pull/10399). When scrolling with the mouse press, after the release a short animation gets executed
In this case the following events are called:
- MousePress
- MouseMove
- MouseRelease -> creates animation
- Exit

For the touchpad only `MouseMove` and `Exit` is called and therefore no animation gets triggered because no MousePress was triggered

### Reproducible Code (if applicable)

```slint
component Item inherits Rectangle {
    height: 60px;
    background: @linear-gradient(0, greenyellow, skyblue);
}

export component L inherits Rectangle {
    width: 600px;
    height: 600px;

    ListView {
        property<length> entry_height: 600px;
        viewport-width: parent.width;
        
        for i in 100: Item { }
    }
}
```

### Environment Details

- Slint Version: bcc1ce0143f0446b465cd0a6572969b72eea986f

### Product Impact

No smooth animation when using a touchpad

**Comment by ogoffart** [comment]
This is the same as https://github.com/slint-ui/slint/issues/4115
